packages/backend/src/persistence/storage/postgres/setAccount.ts

```diff
@@ -0,0 +1,6 @@
+import { PrismaClient } from '@prisma/client';
+import { getOrCreateAccount } from './utils/getOrCreateAccount';
+
+export const setAccount = (db: PrismaClient) => async (ensName: string) => {
+    return getOrCreateAccount(db, ensName);
```

Comment:
You've to add the actual account as an arg. At the moment only empty accounts can be created with getAccount

Reply:
The new setAccount function does not need more arguments.